transaction/core/src/validation/validate.rs

```diff
@@ -65,6 +61,13 @@ pub fn validate<R: RngCore + CryptoRng>(
     // Note: The transaction must not contain a Key Image that has previously been
     // spent. This must be checked outside the enclave.
 
+    // In the 1.2 release, it is planned that clients will know to read memos,
+    // but memos will not be allowed to exist in the chain until the next release.
+    // If we implement "block-version-based protocol evolution, then this function
```

Comment:
Nit: Missing quote. Could also use an MCIP #.
```suggestion
    // If we implement "block-version-based protocol evolution" (MCIP 26), then this function
```